### ISSUE TITLE
verb forms with эбит

### ISSUE THREAD

**Issue by jonorthwash** [apertium/apertium-sah#2]
Sakha has "complex verb forms" with эбит.

> Кэлбит эбиттэр "Они, оказывается, пришли"

Many other Turkic languages have this.

In Turkish, the etymologically identical and semantically similar (`dub` instead of `evid`) form is "gelmiş imişler"/"gelmişmişler":
```
"<gelmişmişler>"
	"gel" v iv past dub
		"i" cop aor p3 pl
```

In Kazakh, it's equivalent morphologically and semantically to "келген екен":
```
"<келген екен>"
	"кел" v iv past evid p3 pl
```
And Kyrgyz
```
"<келген экен>"
	"кел" v iv past evid p3 pl
```

There are a number of strategies for dealing with this sort of construction, and they were discussed to some extent on the apertium-turkic mailing list in 2014.  A Turkic-wide solution should be found.  In the meantime, I'm partial to doing it like Kazakh and Kyrgyz, and fixing everything all at once when a different strategy is decided on.

**Comment by ftyers** [comment]
I agree, it looks more like the Kazakh/Kyrgyz and we're currently working on translation for Kazakh->Sakha, so that solution seems to make most sense for now.

**Comment by varie** [comment]
I tried to add it here - https://github.com/apertium/apertium-sah/commit/f27a04413808f2c55728faf433a17a9e392654e7.
